examples/seq2seq/test_fsmt_bleu_score.py

```diff
@@ -0,0 +1,184 @@
+# coding=utf-8
+# Copyright 2020 Huggingface
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+from sacrebleu import corpus_bleu
+
+from parameterized import parameterized
+from transformers import FSMTForConditionalGeneration, FSMTTokenizer
+from transformers.testing_utils import require_torch, slow, torch_device
+
+
+def calculate_bleu(output_lns, refs_lns, **kwargs) -> dict:
+    """Uses sacrebleu's corpus_bleu implementation."""
+    return {"bleu": round(corpus_bleu(output_lns, [refs_lns], **kwargs).score, 4)}
+
+
+# BLEU eval data was generated using the following code:
```

Comment:
can you put this in a gist and link to it? It's big!

Reply:
I made it into a script, placed in the data dir with the data